tests/helper/helper_telemetry.go

```diff
@@ -0,0 +1,39 @@
+package helper
+
+import (
+	"encoding/json"
+	_ "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-developer/odo/pkg/preference"
+	"github.com/redhat-developer/odo/pkg/segment"
+	"io/ioutil"
+	"os"
+)
+
+//CreateTelemetryDebugFile creates a temp file to use for debugging telemetry.
+//it also sets up envs and cfg for the same
+func CreateTelemetryDebugFile() {
```

Comment:
Do we really need to create the telemetry file before? odo will create it itself

Reply:
Well how do we know the file name then?
The idea is to create a proper file (so we dont have path problems) and use that name as the debug file so we can go back and read it

Reply:
ok

Reply:
This function does more than creating the file, it also sets appropriate env vars. You could name it `EnableTelemetryDebug` or similar to indicate it prepares all of what is needed to have the telemetry debugging